Auth0/Authentication.swift

```diff
@@ -774,15 +774,15 @@ public extension Authentication {
      ```
 
      - parameter credentials: the client's credentials obtained on auth
-     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth.
+     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth
+     - parameter callback: callback with the user's credentials or the cause of the error.
      - important: This method only works for a refresh token obtained after auth with OAuth 2.0 API Authorization.
-     - returns: a request that will either yield an error or Auth0 user's credentials
      */
     func renewExpired(_ credentials: Credentials, scope: String? = nil, callback: @escaping (Error?, Credentials?) -> Void) {
-        guard
-            let refreshToken = credentials.refreshToken,
-            let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60)
-            else { return callback(AuthenticationError(string: "missing refresh_token or have not expired", statusCode: 400), nil) }
+        guard let refreshToken = credentials.refreshToken else {
+            return callback(AuthenticationError(string: "missing refresh_token", statusCode: 400), nil)
+        }
+        guard let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60) else { return callback(nil, credentials) }
```

Comment:
The token should be used right after the user asks for it, so it's up to him if he takes too long and the server rejects it.. Maybe not 5 minutes, but 2 min already is too much. Remember that the problem here is the server, because we can still decode a jwt payload if needed even if expired.. Maybe it's best to check the server "allowed leeway" and pick a smaller value just to be sure. Anyway, it's not too hard to log in again 😛. I'll approve this pr and let you decide if a change it's needed.